main.py

```diff
@@ -0,0 +1,102 @@
+from time import localtime, strftime, sleep
+from colorama import Fore
+import requests
+import random
+import string
+import time
+import os
+
+
+class SapphireGen:
+    def __init__(this, code_type: str, prox=None, codes=None):
+        this.type = code_type
+        this.codes = codes
+        this.proxies = prox
+        this.session = requests.Session()
+        this.prox_api = (
+            "https://raw.githubusercontent.com/TheSpeedX/PROXY-List/master/http.txt"
+        )
+
+    def __proxies__(this):
+        req = this.session.get(this.prox_api).text
+        if req != None:
+            open("./data/proxies.txt", "a+").truncate(0)
+            for proxy in req.split("\n"):
+                proxy = proxy.strip()
+                proxy = f"https://{proxy}"
+                open("./data/proxies.txt", "a").write(f"{proxy}\n")
+
+    def generate(this, scrape=None):
+        if scrape == True:
+            this.__proxies__()
+        else:
+            pass
+
+        os.system("clear")
+        for _ in range(int(this.codes)):
+            try:
+                if this.proxies == True:
+                    prox = {
+                        "https": random.choice(
+                            open("./data/proxies.txt", "r").read().splitlines()
+                        )
+                    }
+                else:
+                    prox = None
+
+                if this.type == "boost":
+                    code = "".join(
+                        [
+                            random.choice(string.ascii_letters + string.digits)
+                            for i in range(24)
+                        ]
+                    )
+                else:
+                    code = "".join(
+                        [
+                            random.choice(string.ascii_letters + string.digits)
+                            for i in range(16)
+                        ]
+                    )
+                req = this.session.get(
+                    f"https://discordapp.com/api/v6/entitlements/gift-codes/{code}",
```

Comment:
i would bump the api version to the latest (9) due to deprecation

Reply:
I could remove /v6, this would by default make it use the current and most intact api version.